src/main/resources/logback.xml

```diff
@@ -9,15 +9,7 @@
         <thread>[ignore]</thread>
         <levelValue>[ignore]</levelValue>
       </fieldNames>
-      <providers>
-        <globalCustomFields>
-          <customFields>{"service":"banner-api"}</customFields>
-        </globalCustomFields>
-        <logLevel>
```

Comment:
The Logback config has changed. The `logLevel` is reported as 'ignored' by Logback and doesn't need to be configured here (so removed it). When testing in a local env, the log level and severity seems to be included in the structured log by default